README.rst

```diff
@@ -165,10 +165,10 @@ Add the following settings to ``settings.py``:
 
     FORM_RENDERER = 'django.forms.renderers.TemplatesSetting'
 
+    ASGI_APPLICATION = "openwisp_controller.geo.channels.routing.channel_routing"
     CHANNEL_LAYERS = {
         'default': {
-            'BACKEND': 'asgiref.inmemory.ChannelLayer',
-            'ROUTING': 'openwisp_controller.geo.channels.routing.channel_routing',
+        "BACKEND": "channels.layers.InMemoryChannelLayer"
```

Comment:
seems the indentation is odd, also keep the same quoting style used in the file, which is using single quotes